website/docs/partials/tfc-package-callouts/ephemeral-workspaces.mdx

```diff
@@ -0,0 +1 @@
+-> **Note:** Ephemeral workspace functionality is available in Terraform Cloud **Plus** Edition. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
```

Comment:
Can we refer to automatic destroy runs in this partial? I feel we keep putting it right next to "Automatic Destroy Runs", but don't clarify how "ephemeral workspaces"  are _related_ to automatic destroy runs.